popde/density_estimate.py

```diff
@@ -1,3 +1,128 @@
+class SimpleGaussianKernelDensityEstimation:
+    """
+    Kernel Density Estimation (KDE) using Gaussian Kernels
```

Comment:
again, if the kernel is not necessarily Gaussian it should not be stated here that it is

Reply:
fix this comment